android-components/components/browser/toolbar/src/main/res/values-ldrtl/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Could you add the Mozilla license here? You can see an example with the `dimens.xml` file below.

Reply:
Good catch, just added!